nucliadb_node/nucliadb_node/indexer.py

```diff
@@ -377,8 +377,21 @@ async def _index_message(self, pb: IndexMessage):
             status = await self._set_resource(pb)
         elif pb.typemessage == TypeMessage.DELETION:
             status = await self._delete_resource(pb)
-
         if status is not None and status.status != OpStatus.Status.OK:
+            if (
+                status.status == OpStatus.Status.ERROR
+                and "Shard not found" in status.detail
```

Comment:
Maybe in the future we should use meaningful status codes.. this seems error prone